presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -88,12 +88,9 @@ Property Name                                      Description
 
 ``hive.storage-format``                            The default file format used when creating new tables.       ``RCBINARY``
 
-``hive.compression-codec``                         The compression codec to use when writing files.             ``GZIP``
```

Comment:
Let's leave the ordering the same: storage format and compression codec are related and should be together.